backend/Hortogram/Hortogram/Business/UserService.cs

```diff
@@ -93,5 +129,68 @@ public bool UpdateUser(Guid id, string firstName, string lastName, string email,
                 return false;
             }
         }
+
+        public List<UserFollowersResponse> GetFollowers(Guid userId)
+        {
+            try
+            {
+                User user = Utils.ConvertUserResponseToUser(GetById(userId));
+
+                if (user == null)
+                    return null;
+
+                List<UserFollowersResponse> followers = UserRepository.GetFollowers(user);
+                return followers;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
Pode apagar o Console.WriteLine, existem alguns por aqui

Reply:
Eu tenho deixado os WriteLine como uma forma de logging enquanto to debugando